USAGE.rst

```diff
@@ -39,26 +39,31 @@ To execute a build pipeline locally, try the following example command:
      python execute_pipeline.py \
        --pipeline_kwargs={\'sleep_secs\':2} SamplePipeline
 
-To execute a build pipeline remotely, start two consoles and run the following command
-in one console:
+Remote
+******
+
+To execute a build pipeline remotely, start two consoles (on the same or different machines) and run 
+the following command in one console:
```

Comment:
Ok I see from Jacob's comment what's going on. These instructions are somewhat unclear about what console is doing what; it seems to be that the first console is the "remote" console.